src/microbe_stage/organelle_unlocks/IUnlockCondition.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace UnlockConstraints
+{
+    using System;
+
+    public interface IUnlockCondition
+    {
+        public bool Satisfied(EventArgs args);
+
+        public void GenerateTooltip(LocalizedStringBuilder builder, EventArgs args);
```

Comment:
This isn't an event so why is `EventArgs` type used here? It'd be better to use an empty interface that both statistics and world and player data classes implement.